docs/reference/api/openai.md

```
@@ -232,6 +232,60 @@ curl http://localhost:30000/v1/models
 
 ---
 
+### Audio Transcriptions
+
+Transcribe an audio file (batch). Requires a worker serving an ASR model.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)